utils/shared/expression_utils.py

```
@@ -3,50 +3,50 @@
-import contextlib
 from datetime import datetime
 from typing import Union, Optional, Any
-from utils.path_resolver import resolve_relative_to_pyt
 
 
 REQUIRED_REGISTRY_KEYS = {"name", "type", "length", "alias", "category", "expr", "oid_default", "orientation_format"}
 
 
-def load_field_registry(registry_path: str, config: Optional[dict] = None, category_filter: Optional[str] = None) \
-        -> dict:
+def load_field_registry(cfg: ConfigManager, category_filter: Optional[str] = None) -> dict:
```

Ruff (0.11.9)

Undefined name `ConfigManager`

(F821)